CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # 0.x.y (2022-aa-bb)
 
+- Add "leaveCommunityFund" functionality to governance canister.
```

Comment:
can you add this under "### Features" or any other related type?

Reply:
Ah, I didn't realize those were for the specific release. Sure.

Reply:
done